pages/api/webhooks/checkout-calculate-taxes.ts

```diff
@@ -21,15 +22,24 @@ const handler: NextApiHandler = async (request, response) => {
       .json({ success: false, message: error.message });
     return;
   }
+  const saleorDomain = request.headers[SALEOR_DOMAIN_HEADER];
 
   const body: CheckoutPayload[] =
     typeof request.body === "string" ? JSON.parse(request.body) : request.body;
 
   const checkoutPayload: CheckoutPayload = body[0];
 
-  // FIXME: this part of settings will be fetched from App.metadata and defined based
-  // on channnel used in checkout.
-  const taxJarConfig = getTaxJarConfig();
+  const taxJarConfig = await getTaxJarConfig(
+    saleorDomain as string,
+    checkoutPayload.channel.slug
+  );
+  if (!taxJarConfig) {
+    response
+      .status(404)
+      .json({ success: false, message: "TaxJar is not configured." });
+    console.log("TaxJar is not configured.");
```

Comment:
```suggestion
    console.error("TaxJar is not configured.");
```
This makes it easier to debug issues in production - for example, Vercel will list errors in a separate tab:

![CleanShot 2022-08-01 at 12 47 22@2x](https://user-images.githubusercontent.com/4144459/182132205-16d616af-bde9-4963-803a-ce5490e4904c.png)